src/helpers.js

```diff
@@ -117,25 +119,88 @@ export function isTraversable(tileID, me) {
   }
 }
 
+/* eslint no-param-reassign: ["error", { "ignorePropertyModificationsFor": ["bigGrid"] }] */
+
+/*
+ * place all values from smallGrid into bigGrid. Align the upper left corner at x, y
+ */
+export function fillGridWithSubgrid(bigGrid, smallGrid, x, y) {
+  const smallGridWidth = smallGrid.length;
+  const smallGridHeight = smallGrid[0].length;
+
+  for (let i = 0; i < smallGridWidth; i++) {
+    for (let j = 0; j < smallGridHeight; j++) {
+      bigGrid[i + x][j + y] = smallGrid[i][j];
+    }
+  }
+}
+
+
+/* Returns a 2d cell array of traversible (1) and blocked (0) cells inside a tile.
+ *
+ * tileIsTraversable: if this tile is traversable
+ * cpt: number of cells per tile
+ * objRadius: radius of untraversable object in pixels
+ */
+export function traversableCellsInTile(tileIsTraversable, cpt, objRadius) {
+  const gridTile = [];
+  // Start with all traversable
+  let i;
+  let j;
+  for (i = 0; i < cpt; i++) {
+    gridTile[i] = new Array(cpt);
+    for (j = 0; j < cpt; j++) {
+      gridTile[i][j] = 1;
+    }
+  }
+
+  if (!tileIsTraversable) {
```

Comment:
I believe the appropriate hotfix is to use the >20*sqrt(2) radius and raise an issue as that will create functional equivalence to past behavior and allow us to move forward with other development. With that change I'd be ready to approve this.